index.js

```diff
@@ -49,7 +47,7 @@ getCharacters();
 //add event listener
 
 nextButton.addEventListener("click", () => {
-  if (page < maxPage) {
+  if (page < 42) {
```

Comment:
Very good. Please use `maxPage` here instead of a hardcoded value.
```suggestion
  if (page < maxPage) {
```